rules/scala_proto/private/ScalaProtoWorker.scala

```diff
@@ -91,6 +94,7 @@ object ScalaProtoWorker extends WorkerMain[Unit] {
     if (exitCode != 0) {
       throw new AnnexWorkerError(exitCode)
     }
+    InterruptUtil.throwIfInterrupted()
```

Comment:
I went back and forth on this, but I did this so we'd return a cancelled response rather than a normal one. Either way works for the Bazel server. I didn't feel strongly about these and can remove them if we want.

Reply:
I don't feel strongly about them either; I was just wondering. I'm fine with keeping them.